apollo-training/package.json

```diff
@@ -36,6 +39,7 @@
     "@babel/core": "^7.9.0",
     "@babel/node": "^7.8.7",
     "@babel/preset-env": "^7.9.0",
+    "eslint": "^7.32.0",
     "nodemon": "^2.0.2"
```

Comment:
What is eslint and Why (**^**) is used?

Reply:
eslint (ecma script lint) is use for following Airbnb style to lint code,.
and ^ carret user for diplay version info in package.json for “Approximately equivalent to version”, gives backwards-compatible new functionality as well.